allennlp/models/semantic_role_labeler.py

```diff
@@ -51,6 +55,8 @@ def __init__(self, vocab: Vocabulary,
         self.stacked_encoder = stacked_encoder
         self.tag_projection_layer = TimeDistributed(Linear(self.stacked_encoder.get_output_dim(),
                                                            self.num_classes))
+        initializer(self.stacked_encoder)
```

Comment:
The trouble is if someone passes in a `TextEmbedder` that uses a character-level encoder - the encoder won't get initialized.  It'd be nice to have some way for modules to say "I've already been initialized", so that the embeddings don't get messed up, but everything else works.